### PR TITLE
Update ARIES_AUTORENEWAL_RFP.md

### PR DESCRIPTION
**Pull request by nfaulknerAK**
move from draft to final RFP

### CODE REVIEW

File: ARIES_AUTORENEWAL_RFP.md [hunk 1]
```diff
@@ -510,20 +509,19 @@ schedule, such as the deadline for receipt of proposals, is delayed, the
 rest of the schedule may be shifted by the same number of days. **All
 times are Alaska Prevailing Time**.
 
--   Issue RFP on TBD
+-   Issue RFP on November 27, 2021
```
Comment (dvvarela) [inline ARIES_AUTORENEWAL_RFP.md:512]: s/b 16th

File: ARIES_AUTORENEWAL_RFP.md [hunk 2]
```diff
@@ -510,20 +509,19 @@ schedule, such as the deadline for receipt of proposals, is delayed, the
...
 
--   Deadline for receipt of questions on + 10 days
+-   Deadline for receipt of questions on Saturday November 16, 2021 4PM
```
Comment (dvvarela) [inline ARIES_AUTORENEWAL_RFP.md:514]: s/b November 27th?

File: ARIES_AUTORENEWAL_RFP.md (hunk 1 above)
Comment (nfaulknerAK) [reply ARIES_AUTORENEWAL_RFP.md]: fixed

File: ARIES_AUTORENEWAL_RFP.md (hunk 2 above)
Comment (nfaulknerAK) [reply ARIES_AUTORENEWAL_RFP.md]: fixed

File: ARIES_AUTORENEWAL_RFP.md [hunk 3]
```diff
@@ -868,8 +866,8 @@ in the last five (5) years in a technical leadership role.
 
 ## Sec. 3.04 Contract Term and Work Schedule
 
-The term of this contract will be from approximately December 15, 2021
--- September 30, 2022. Any potential for contract amendments that alter
+The term of this contract will be from approximately January 3, 2021
```
Comment (dvvarela) [inline ARIES_AUTORENEWAL_RFP.md:869]: Should be = to Contract Start? January 10, 2022

Comment (nfaulknerAK) [reply ARIES_AUTORENEWAL_RFP.md]: fixed

**Review by dvvarela — APPROVED** [review]

**Pull request merged**